syntax="proto2";

package aap_protobuf.service.control.message;

message PingRequest {
    required int64 timestamp = 1;
    optional bool bug_report = 2;
    optional bytes data = 3;
}